compute_endpoint/globus_compute_endpoint/exception_handling.py

```diff
@@ -86,6 +87,21 @@ def wrapper(*args, **kwargs):
                 message:           {e.text}
                 """
             )
+
+            # This specific Auth error has a common cause
+            if e.http_status == 400 and e.code == "Error":
+                try:
+                    error_info = json.loads(e.text)
+                    if "invalid_grant" == error_info.get("error"):
+                        msg += (
+                            "\nYour Globus Compute credentials might have expired."
+                            " You can clear them with `globus-compute-endpoint logout`"
+                            " before retrying the command."
```

Comment:
- Given at least one interaction, you might consider adding a note that `globus-compute` credentials are not shared with the `globus` (SDK) credentials.
- Grammar meizer: `can` -> `may`

Reply:
Hmm can vs. may is a debatable point.  I'm thinking 'can' is 'this is a way that it can be done'.  'may' sounds more like 'you might be able to' or 'you might want to'.  But +1 to the opportunity to clarify the 'different creds' point.

Reply:
Another tact: take the `you` out entirely.  Consider:
```
The Globus Compute credentials may have expired.  Use `logout` to clear them.
```

No need to reference "retrying" the command -- the user will likely do that in due course of "WTH?!  It failed ... oh, what's this error message?  Huh."

Reply:
New message:

``Your Globus Compute credentials might have expired. You can clear them with `globus-compute-endpoint logout` before retrying the command.
Note that Globus Connect (Transfer) credentials is managed  via the `globus` command, separately from the Globus  Compute ones.``